imap.py

```diff
@@ -116,7 +121,13 @@ def get_mails(cls, servers):
                         server.mailbox)
                     if new_mail:
                         mails[server.id].append(new_mail)
-                imapper = cls.connect(server)
+
+                result = None
+                try:
+                    imapper = cls.connect(server)
```

Comment:
`server.action_after(imapper...)`

must be inside try:

Reply:
6a77891f5f83ea6cdd3727ecfade5afbec87c64d